### ISSUE TITLE
Commands for adding/removing files from a project

### ISSUE THREAD

**Issue by StefanUPB** [sonata-nfv/tng-sdk-project#8]
Similar to git, `tng-project add <file>` should add the specified file to project.yaml, so it's packaged by the packager. 
Vice versa, `tng-project reset/remove` removes it from project.yaml, so it is ignored by the packager

**Comment by mpeuster** [comment]
We might need developers to tell us what file types they add.
(we could try to autodetect based on file endings where possible)

```
tng-project add -t application/binary folder/folder/myvnf.qcow2
```

**Comment by StefanUPB** [comment]
[python-magic](https://github.com/ahupp/python-magic) can detect MIME types without fileendings. But it just returns `text/plain` for all `.yml` files as it can't distinguish between VNFDs and NSDs.

I guess, we'll have to build something ourselves here or simply require users to specify what they are adding. Maybe using a few different flags like `tng-project add --vnfd sample.yml` or `--image`.

**Comment by mpeuster** [comment]
Yes that sounds good. I guess we need a combination:

- python-magic
- file ending
- if yaml file, check if it is an NSD, VNFD, or whatever descriptors we have

with this three things we should be able to detect most things used in packages. and if they do not work, a manual fallback can be used.

**Comment by StefanUPB** [comment]
Extension for later: Optional `--tag` option that allows to add tags to be used in the [NAPD](https://github.com/sonata-nfv/tng-schema/wiki/PkgSpec_LATEST#file-napdyaml-nfv-advanced-package-descriptor)

**Comment by StefanUPB** [comment]
Optional `--type` option to set the MIME type manually. This might be necessary when we can't detect the file type (e.g., of OSM descriptors). In this case we should throw a warning, use a best-effort categorization, and ask the user to provide the type manually.

**Comment by StefanUPB** [comment]
We planned to determine the MIME/type of descriptors based on their `descriptor_schema`, e.g., `https://raw.githubusercontent.com/sonata-nfv/tng-schema/master/service-descriptor/nsd-schema.yml`.

Should we try to guess the type by slicing this URL (e.g., into `nsd-schema`)? But this isn't very clean and breaks if we move or rename the schema.

Alternatively, we could add the MIME/type directly in the descriptors such that it can be read directly instead of `descriptor_schema`.

Also, what about Sonata/Tango yaml files without `descriptor_schema`? For example, FSMs and SSMs don't even seem to have schemas. And what are "patterns"? Do we need all this in Tango?

**Comment by StefanUPB** [comment]
- Include a mapping from schema URLs to MIME/types in a configuration file per workspace.
- Remove folders and samples for FSMs, SSMs, patterns. These are old Sonata artifacts (now deprecated).
- Update sample NSD and VNFD

**Comment by StefanUPB** [comment]
Closed by PR #12
